tests/common/snappi_tests/snappi_test_params.py

```diff
@@ -24,17 +25,18 @@ def __init__(self):
             test_tx_frames (list): number of test frames transmitted for priorities to test ex. [2000, 3000]
                 for ex. priorities 3 and 4
             gen_background_traffic (bool): whether or not to generate background traffic (default: True)
-            pause_flow_params (dict): pause frame parameters
-                Params:
-                    pause_frame_size (int): pause frame size in bytes (default: 64)
-                    pause_frame_rate (int): pause frame rate in frames per second
-                                            (default: pause dur to block link fully)
-                    pause_flow_dur (int): pause flow duration in seconds (default: -5 to signal continuous mode)
-                    pause_flow_delay (int): pause flow delay in seconds (default: 0)
-                    link_blockage_threshold (int): link blockage threshold in number of overlaps per pause dur
-                                                   (default: 2)
             poll_device_runtime (bool): whether or not to poll the device for stats when traffic is running
                                         (default: False)
+                for priorities 3 and 4
+            gen_background_traffic (bool): whether or not to generate background traffic (default: True)
+            ecn_params (dict): ECN parameters
+                Dict values:
+                    kmin: minimum ECN marking threshold
+                    kmax: maximum ECN marking threshold
+                    pmax: maximum ECN marking probability
+            pkt_size (int): packet size for background and test traffic
```

Comment:
It seems that pkt_size and pkt_count are not parameters of SnappiTestParams class

Reply:
let me remove this here - its redundant